stty.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+// GetTerminalSize using stty
+func GetTerminalSize() (int, int) {
+	cmd := exec.Command("stty", "size")
```

Comment:
Is there a reason `stty` is needed? I'd much rather not have to shell out if possible. You can get the screen size using `game.Screen().Size()`, but I think the problem when I was experimenting previously was that it returned zero(s) until the first time the screen was drawn.

It looks like termloop uses [`termbox.Size()`](https://github.com/JoelOtter/termloop/blob/bf4ace9ee910543f49f13717d4ceac223343d861/game.go#L36) internally. Maybe a better interim solution would be to call termbox instead of `stty`.

Ultimately, I think this is a good candidate for a change to termloop if possible. I don't think there's a technical reason to not do this - it was previously discussed in JoelOtter/termloop#30 but the discussion dropped off. cc @JoelOtter

Reply:
Ah yeah, that was the api I was looking for. Much cleaner! Had to initialize Termbox before getting the size though, hopefully this can be fixed in Termloop.